config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml

```diff
@@ -119,6 +119,39 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
+  - name: pull-kubernetes-e2e-gce-alpha-features
+    always_run: false
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-bazel-scratch-dir: "true"
+    spec:
+      containers:
+      - args:
+        - --root=/go/src
+        - --repo=k8s.io/kubernetes=$(PULL_REFS)
```

Comment:
ah, also add `--repo=k8s.io/release` so you can stage your local build.

Reply:
Thanks! Created a fix PR at: https://github.com/kubernetes/test-infra/pull/9957